test/pageobjects/SelfieIntro.js

```diff
@@ -0,0 +1,22 @@
+import BasePage from './BasePage.js'
+import { verifyElementCopy } from '../utils/mochaw'
+
+class SelfieIntro extends BasePage {
+  get selfieIcon() { return this.$('.onfido-sdk-ui-Selfie-selfieIcon')}
+  get glassesIcon() { return this.$('.onfido-sdk-ui-Selfie-glassesIcon')}
+  get continueButton() { return this.$('.onfido-sdk-ui-Button-button-text')}
```

Comment:
To avoid flaky tests, I would suggest using `async continueButton() { return this.waitAndFind('.onfido-sdk-ui-Button-button-text')}` instead

Reply:
I tested this out and the ui test is no longer able to find the continue button. Also why do we need an async on the button and not on the selfie or glasses icon ? These are loaded together

Reply:
I suggested using `async` + `waitAndFind` on `continueButton()` because I thought it would be the first element that you are trying to find in the new screen, but because you are using `verifyUIElementsOnTheSelfieIntroScreen` which already  includes `waitAndFind` inside `this.title()` then you should be fine. Although, tests should not fail if you use `async` + `waitAndFind` on `continueButton()`.